goalie/go_mesh_seq.py

```diff
@@ -64,9 +64,7 @@ def get_enriched_mesh_seq(
             qoi_type=self.qoi_type,
             parameters=self.params,
         )
-
-        # Update function spaces for enriched mesh
-        mesh_seq_e.update_function_spaces
```

Comment:
Note that I prefer not to add comments that are already clear from the code / method name.

Reply:
I'm writing some [Development Practices](https://github.com/pyroteus/movement/wiki/Development-Practices) over in the Movement wiki and I'll put this in there.

Reply:
@jwallwork23 - Noted. I have a tendency to over-comment and will work on it especially when the attempt is to make the method name self explanatory.